react-state3/react-state-3_journal-app-form/src/components/EntryForm/index.js

```diff
@@ -1,18 +1,57 @@
 import "./EntryForm.css";
 import Button from "../Button";
+import { useState } from "react";
+
+export default function EntryForm({ onAddEntry }) {
+  const [formData, setFormData] = useState({ motto: "", notes: "" });
```

Comment:
Do we need a second state here? For simplicity you can use just one state in the `App` component.